GitUI/Translation/English.Plugins.xlf

```diff
@@ -413,65 +413,9 @@ Are you sure you want to continue?</source>
       </trans-unit>
     </body>
   </file>
-  <file datatype="plaintext" original="FindLargeFilesForm" source-language="en">
```

Comment:
I guess this goes too far. 🤔 These translations seem to have got lost.

Reply:
Not a new problem it seems, the texts are not in English.xlf either...

Reply:
The type is getting collected:
![image](https://user-images.githubusercontent.com/4403806/209935518-c6dcc8bb-789c-4d56-bb4a-5784eeb580f1.png)

...however, it can't be instantiated for the translations to get processed:
![image](https://user-images.githubusercontent.com/4403806/209935749-68299b89-b955-4654-92fb-b382b7202521.png)